css-cascade-4/Overview.bs

```diff
@@ -221,14 +221,12 @@ Conditional ''@import'' Rules</h3>
 	(even if it is loaded through some other link).
 
 	<div class="example">
-		The following rule illustrates how an author can provide fallback rules for legacy user agents
-		without impacting network performance on newer user agents:
+		The following rule illustrates how an author can provide rules for modern user agents
+		which support newer web features and without impacting network performance on unsupported user agents.
+		In this example, a CSS file which loads COLRv1 fonts is only fetched when it are supported:
```

Comment:
A few grammar fixes and rewrapping per https://rhodesmill.org/brandon/2012/one-sentence-per-line/ :)

```suggestion
		The following rule illustrates how an author can provide rules
		for modern user agents that support newer web features 
		without impacting network performance on older user agents.
		In this example, a CSS file which loads COLRv1 fonts
		is only fetched when COLRv1 is supported:
```

Reply:
Fixed with suggested change, thanks!